src/main/proto/service.proto

```diff
@@ -69,3 +71,14 @@ message CreateCustomerRoleRequest {
 message CreateCustomerRoleResponse {
   optional bytes role_id = 1;
 }
+
+message MoveAccountBalanceRequest {
+  optional bytes from_account_id = 1;
+  optional bytes to_account_id = 2;
+  optional string amount = 3;
+}
+
+message MoveAccountBalanceResponse {
+  optional string from_account_id_balance = 1;
+  optional string to_account_id_balance = 2;
```

Comment:
IMO an `{account_id: balance}` map would be more useful here, but not a blocking comment.

Reply:
Since there's only 2, then maybe it's best to keep it simple? As far as how clients interact with this, if they use something like the Java gRPC client, it might just be simpler to do something like `response.getFromAcccountIdBalance` then maybe something like `response.get<map name>.get(fromAccountId)`. Not sure if this is even a correct interpretation/understanding of how maps in gRPC work and how the client would use them.

Reply:
Sure, up to you. :)